components/CollectionCells/src/MDCCollectionViewCell.m

```diff
@@ -424,16 +424,32 @@ - (void)updateInterfaceForEditing {
 
 #pragma mark - Selecting
 
++ (NSBundle *)bundleWithName:(NSString *)bundleName {
+
+  NSBundle *baseBundle = [NSBundle bundleForClass:[self class]];
+  NSString *bundlePath = [baseBundle pathForResource:bundleName ofType:@"bundle"];
+  NSBundle *bundle = [NSBundle bundleWithPath:bundlePath];
+  NSAssert(bundle, @"Missing bundle %@.", bundleName);
+
+  return bundle;
+}
+
 - (void)setSelected:(BOOL)selected {
   [super setSelected:selected];
   if (selected) {
     _editingSelectorImageView.image =
-        [UIImage imageWithContentsOfFile:[MDCIcons pathFor_ic_check_circle]];
+                  [UIImage imageNamed:@"ic_check_circle"
+                             inBundle:[MDCCollectionViewCell
+                                          bundleWithName:@"MaterialIcons_ic_check_circle"]
+        compatibleWithTraitCollection:nil];
     _editingSelectorImageView.tintColor = self.editingSelectorColor;
     self.accessibilityTraits |= UIAccessibilityTraitSelected;
   } else {
     _editingSelectorImageView.image =
-        [UIImage imageWithContentsOfFile:[MDCIcons pathFor_ic_radio_button_unchecked]];
+    [UIImage imageNamed:@"ic_radio_button_unchecked"
+               inBundle:[MDCCollectionViewCell
+                         bundleWithName:@"MaterialIcons_ic_radio_button_unchecked"]
+compatibleWithTraitCollection:nil];
```

Comment:
Formatting.
Since compatibleWIthTraitCollection: won't fit, align the ':' for the other parameters, then indent this one 4 space past the original "[UIImage imageName:..."